doc/kci_testsuite.md

```diff
@@ -0,0 +1,289 @@
+### Verify your requirement:
+
+-  First validate whether you really need to create a new rootfs images? you find existing rootfs variants using
+  `kci_rootfs list_variants`. If those matches your requirement, please make use of it. 
+
+-  If you want to make minor modification. Then check below options:
+     - If you need to install specific Debian package, have look at `extra_packages` option in `rootfs-configs.yaml` file.
+     - To install specific test suite built from source, pep3 etc have a look at `script` option in `rootfs-configs.yml` file. It allows to you
+        run prepare rootfs image in a desired manner.
+     - To install specific files or helper scripts on specific location, have a look at `overlay` option in `rootfs-configs.yml` file.
+
+
+### How to add test suite to a rootfs image
+
+1. Let's understand how test suite is added to a rootfs image. For this, we will explore `buster-v4l2` rootfs image.
+
+2. Take a look at `rootfs-configs.yaml` for `buster-v4l2`. It should display following config entries.
+
+
+```
+  buster-v4l2:
+    rootfs_type: debos
+    debian_release: buster
+    arch_list:
+      - amd64
+      - arm64
+      - armhf
+    extra_packages_remove:
+      - bash
+      - e2fslibs
+      - e2fsprogs
+    script: "scripts/buster-v4l2.sh"
+    test_overlay: "overlays/v4l2"
+```
+
+
+3. We will look into last three config entries namely, `extra_packages_remove`, `script` and `test_overlay`.
+
+   `extra_packages_remove` as name suggests it will remove the listed packages from final rootfs image. In this case,
+    packages like bash, e2fslibs and e2fsprogs are deleted from the image.
+
+4. Understand that `script` path is relative to `kci_rootfs build --data-path` value. By default its `jenkins/debian/debos/script`
+   Lets have quick look at its current contents:
+
+```
+$ ls jenkins/debian/debos/scripts/
+
+buster-cros-ec-tests.sh  buster-igt.sh  buster-v4l2.sh  create_initrd_ramdisk.sh  create_manifest.py  crush.sh  nothing.sh  setup-networking.sh  stretch-ltp.sh  strip.sh
```

Comment:
`ls -l` or `ls -1` to show them in a vertical list would be easier to read than a very long line.